### ISSUE TITLE
Obfuscated userscript

### ISSUE THREAD

**Issue by 0x464e** [pog5/nealpasswordgame#4]
Hi, why would you have obfuscated the userscript?  
Not only does it make the script seem very suspicious but also greatly hinders the ability of someone else fixing/contributing to it.

I just now I found this project and it seemed fun and cool, I tried the script, but it's broken. I could've fixed it, but I'm not willing to touch that minified javascript.

**Comment by P4jMepR** [comment]
It's just minified, it's not that hard to prettify it tho ;)

**Comment by 0x464e** [comment]
> It's just minified, it's not that hard to prettify it tho ;)

Yes, minified as I said. And minification is obfuscation...
And sure it can be "prettified" but minification is a lossy process and the result is garbage is any case.. And why would that be needed when the original source code could and should just be available.  
Anyway, thanks for the useless comment.

**Comment by pog5** [comment]
It's not obfuscated, It's minified. It's done to save on space. I have linked to the original source code used to build it in both the main page, cheats and in the release notes

**Comment by pog5** [comment]
Build the userscript yourself if you don't trust me that much :))

**Comment by pog5** [comment]
> Hi, why would you have obfuscated the userscript? Not only does it make the script seem very suspicious but also greatly hinders the ability of someone else fixing/contributing to it.
> 
> I just now I found this project and it seemed fun and cool, I tried the script, but it's broken. I could've fixed it, but I'm not willing to touch that minified javascript.

Also I just fixed the moon phases being broken. If you cared about it that much and didn't want to just hate on the project you could have opened a issue or just read the release notes where the source was linked.

**Comment by 0x464e** [comment]
Ok, I now finally found it. Seems it only in the release notes, and as a easily missable link.  
I never even know it's not your code, you're just distributing someone else's project with almost no credit. Of course I wouldn't have posted an issue to this repo if I knew the actual project had its own different repo and owner.

And minifying is a form of obfuscation, as we established already. And I don't understand how that's supposed to some important point anyway. 

And why would I open an issue about moon phases being broken, when there was already a pull request for fixing it. And moon phase wasn't my issue anyway, there still is some other error causing the script to crash. But I couldn't really see what it was due to the obfuscated script. But the crashing wasn't what my issue was about anyway. My issue was me thinking you're denying access to the userscript's source code.

Anyway, thanks, now I know the TAS project doesn't even belong to this repo, so I won't waste any more of your time.
